install-deps

```diff
@@ -8,25 +8,42 @@ set -e
 
 install_openblas() {
     # Get and build OpenBlas (Torch is much better with a decent Blas)
-    cd /tmp/
-    rm -rf OpenBLAS
-    git clone https://github.com/xianyi/OpenBLAS.git
-    cd OpenBLAS
-    if [ $(getconf _NPROCESSORS_CONF) == 1 ]; then
-        make NO_AFFINITY=1 USE_OPENMP=0 USE_THREAD=0
+    # Optionally set environment variable PREFIX to control
+    # the installation directory.
+
+
+    local PATH=$PATH:/sbin ## improve chances of finding ldconfig
+    # Only proceed installing OpenBLAS if either ldconfig is unavailable, or ldconfig
+    # reports that OpenBLAS is not already installed.
+    if ! type ldconfig >/dev/null || ! ldconfig -p | grep -q libblas.so; then
```

Comment:
I have now made a change to accept both libblas.so and libopenblas.so:
`if ! type ldconfig >/dev/null || ! ldconfig -p | grep -q "lib\(open\)\?blas.so"; then`